lib/glass_octopus/middleware/avro_parser.rb

```diff
@@ -0,0 +1,32 @@
+require 'avro_turf/messaging'
```

Comment:
Please handle `LoadError`s and print a user/developer friendly message to saying which gem should be installed. Other middleware already do this. You could adjust the messaging for this middleware.